hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.less

```diff
@@ -79,4 +79,16 @@
 
 .ant-drawer-open .ant-drawer-mask {
   opacity: 0 !important;
+}
+
+div.svg-container {
+  width: 1400px !important;
```

Comment:
Is hardcoding this width here compatible with all displays and large displays ? Shouldn't this be relative to screen size ?

Reply:
@devmadhuu 
We have tested different screensize and Responsive web also. Attached new screenshots with testing result.

Reply:
why is this hardcoded value will be good? what happens if you are checking it on smaller displays?